helm-chart/values.yaml.changelog.md

```diff
@@ -5,6 +5,36 @@ For changes that require manual steps other than changing values, please check o
 Please follow this convention when adding a new row
 * `<type: NEW|EDIT|DELETE> - *<resource name>*: <details>`
 
+## Upgrading to Renku 0.44.0
+
+The encryption keys used by the Webhook service and Token Repository have been moved around and now they do not have to
+be base64 encoded in the values file anymore. Although the Helm chart will take care of generating a new secret in the
+correct form from the old one (for existing deployment being upgraded), we suggest to also store those secrets in the
+value file.
+
+To cleanup the old secrets, please remove the following fields:
+
+* DELETE `graph.tokenRepository.tokenEncryption.secret`
+* DELETE `graph.webhookService.hookToken.secret`
+
+Then extract the content of the old secrets for the Token Repository service using `kubectl`:
+
+```shell
+kubectl -n renku get secrets renku-token-repository -o jsonpath="{.data['tokenRepository-tokenEncryption-secret']}" | base64 -d | base64 -d
```

Comment:
The secrets are base64 encoded twice? This is really confusing.

Reply:
Alas they are (you can verify this yourself by issuing the above command in any cluster). This is one of the point of this PR: to fix this tech debt that we carried around for a while...

Reply:
I know, that was one of the reasons we decided to do the change.